big_tests/tests/mod_ping_SUITE.erl

```diff
@@ -161,3 +187,13 @@ wait_for_ping_req(Alice) ->
     <<"urn:xmpp:ping">> = exml_query:path(PingReq, [{element, <<"ping">>},
                                                     {attr, <<"xmlns">>}]),
     PingReq.
+
+no_changes_in_metrics(Domain) ->
```

Comment:
I don't see this function being used anywhere. Is it a leftover?

Reply:
Yes, it is. I could have sworn I didn't push it...